src/test/java/com/wikia/webdriver/testcases/logintests/LoginTests.java

```diff
@@ -4,29 +4,35 @@
 import com.wikia.webdriver.common.core.configuration.Configuration;
 import com.wikia.webdriver.common.properties.Credentials;
 import com.wikia.webdriver.common.templates.NewTestTemplate;
+import com.wikia.webdriver.pageobjectsfactory.componentobject.AuthModal;
 import com.wikia.webdriver.pageobjectsfactory.componentobject.dropdowncomponentobject.DropDownComponentObject;
+import com.wikia.webdriver.pageobjectsfactory.componentobject.global_navitagtion.NavigationBar;
 import com.wikia.webdriver.pageobjectsfactory.pageobject.WikiBasePageObject;
 import com.wikia.webdriver.pageobjectsfactory.pageobject.special.login.SpecialUserLoginPageObject;
 
+import junit.framework.Assert;
 import org.testng.annotations.Test;
 
 /**
  * @author Karol 'kkarolk' Kujawiak <p/> 1. Login user using Special:UserLogin page, 2. Login user
  *         using drop-down 3. Login staff user using Special:UserLogin page, 4. Login staff user
```

Comment:
Please make sure the documentation is up to date, as the test case 3 was changed